sycl/test/extensions/properties/properties_ctor_negative.cpp

```diff
@@ -30,59 +28,56 @@ int main() {
       sycl::ext::oneapi::experimental::boo<int>,
       sycl::ext::oneapi::experimental::bar,
       sycl::ext::oneapi::experimental::fir(3.14, false));
-  // expected-error-re@sycl/ext/oneapi/properties/properties.hpp:* {{static assertion failed due to requirement {{.+}}: One or more property argument is not a property in the property list.}}
+  // expected-error@+2 {{excess elements in struct initializer}}
   sycl::ext::oneapi::experimental::empty_properties_t InvalidPropertyList7{
       sycl::ext::oneapi::experimental::foo{0}};
-  /*
-  // TODO: Uncomment these lines once https://github.com/intel/llvm/issues/13677
-  // has been fixed.
```

Comment:
Suppose https://github.com/intel/llvm/issues/13677 can be closed, even if it's not necessarily fixed? 🤔